pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -446,6 +446,264 @@
         "alpine.3.4.3-x64": {
             "#import": [ "alpine.3.4.3", "alpine.3-x64" ]
         },
+
+    //************************************************
+    //******************CORERT************************
+    //************************************************
+
+        "corert": {
```

Comment:
win*-corert looks like aot. There's but one difference. There is no win-corert, it starts with win7-corert. other than that it's a copy of the aot graph.